### ISSUE TITLE
FR: option to hide the match template text from Note Path links

### ISSUE THREAD

**Issue by avh4** [mdelobelle/obsidian_supercharged_links#95]
(Note: I'm not familiar with Obsidian's plugin API, so maybe implementing this isn't even possible?)

I have a folder `People/` for notes about people, and a Supercharged Links selector for "the path of the note starts with People/", which I style by prefixing an emoji.  The end result is something like "**👤 People/JoJo Siwa**".

It would be great to have an option for the selector that would make it strip out the "People/" prefix when rendering the links so that I could make the end result be "**👤 JoJo Siwa**".  An alternative to stripping it out might be to put a `<span>` with a class around it so it could be hidden (or possibly styled differently) via CSS.

Personally I only care about having this for "Starts with this value" Note Path selectors, but it conceivably could also make sense for other matching types.

**Comment by HEmile** [comment]
I'm not so sure what you mean! If you have a folder called People in which there is a note called JoJo Siwa.md, you can use a wikilink to [[JoJo Siwa]] that refers to People/JoJo Siwa.md. Unless you have JoJo Siwa notes in multiple folders...

The "the path of the note starts with People/" looks at the resulting path of the note, not what it looks like in the wikilink. So wikilinks to [[JoJo Siwa]] will also be styled with the emoji.

**Comment by avh4** [comment]
Yes, as you noted, this would only be for the case where the source text has the folder prefix in the link.  I guess "People/" is a bad example of that one.  A better example would be "Personal/" and "Reference/" where I have an "Education" or "Housing" note under both folders.

**Comment by HEmile** [comment]
I see, yes in that case it is relevant. I'm of the personal and somewhat biased opinion that file names should be unique, but I imagine that's not how it is in every use case. 

I'm not entirely sure if this is something for Supercharged Links, which only aims to add styling, while what you're suggesting actually requires changing the DOM. More importantly, this problem can easily be solved with aliases https://help.obsidian.md/How+to/Add+aliases+to+note , which I think is a more reliable way anyways.
